### DISCUSSION TITLE
"last idea"

### DISCUSSION THREAD

**Discussion by ghost** [StevenBlack/hosts#2665] [Q&A] [answered]

**Comment by StevenBlack** [comment, marked as answer]
Thank you for this question @stokebreakup.

I'll share why this isn't an exciting idea.  It would bypass a key step in workflow. For every release, I check and review all the diffs from each curated source of hosts. If I spot something questionable, I'll either delay the release and raise questions upstream, or not include that source's latest in the release.

A second reason is, presently we can "pause" updates from upstream for individual sources.  For example `adaway.org` and `mvps.org` have updates paused for a variety of reasons, including abandonment, ambiguous control, and other factors.

But ultimately, we currently don't have any of the problems that submodules might resolve.

**Comment by StevenBlack** [comment]
@stokebreakup here are other reasons why submodules aren't an exciting idea for this repository.

Julia Evans just released this interesting git submodule explainer: 

![git_submodules](https://github.com/StevenBlack/hosts/assets/80144/453f3275-b516-463d-8f4d-e0fd5a194750)
